Cargo.toml

```diff
@@ -110,6 +110,8 @@ lazy_static = "1.4"
 meter-core = { git = "https://github.com/GreptimeTeam/greptime-meter.git", rev = "80b72716dcde47ec4161478416a5c6c21343364d" }
 mockall = "0.11.4"
 moka = "0.12"
+notify = "6.1"
+notify-debouncer-full = "0.3"
```

Comment:
If the dependency goes complex here, we can drop the debouncer and just reload some more times. Not a big deal.

Reply:
Yes, for TLS certificates reload feature, I didn't include debouncer intentionally to reduce the dependency tree

Reply:
OK. Let me try to reduce it here also. It should be easy to add back once we find it valuable.